docs/awsmp/provisioning-via-AWS-CloudFormation.md

```diff
@@ -0,0 +1,87 @@
+# EPAM Syndicate Rightsizer provisioning via AWS CloudFormation
+
+AWS CloudFormation is a recommended way to provision EPAM Syndicate Rightsizer instance. 
+It provides ability to specify start parameters for the instance and make it clear and easy to get the configuration outcome.
+
+Please follow these steps to get the ready-to-use product.
+
+## 1. AWS Marketplace Subscription
+Subscribe to EPAM Syndicate Rightsizer in AWS Marketplace. Once done you will be redirected to AWS CloudFormation Console 
+where the following list of parameters should be specified.
+
+## 2. Specify AWS CloudFormation stack parameters
+
+Required parameters:
+- **AWS Subnet id:** сhoose a subnet to launch the instance in;
+- **AWS Security groups ids:** specify security groups to be attached to the instance. By default, 
+the following ports are used: 80 (Defect dojo), 8085 (API), 9000 (Minio API).
+Consider to select Security Group with the ports opened if these services should be available outside the instance;
+- **EC2 Instance Type:** choose a type for the instance;
+- **SSH Key pair name:** choose key pair. Make sure you have access to the selected key. 
+
+Optional parameters:
+- **Instance Name:** name for the instance. Stack name will be used as instance name if custom value is not specified;
+- **IAM Role Name:** name of AWS IAM Role to be attached to the instance. The role is used by EPAM Syndicate Rightsizer to access accounts resources.
+  Keep the field empty if you do not want any role to be attached;
```

Comment:
r8s won't work without attached role
even instance state in CF stack won't be updated